### ISSUE TITLE
Thonny Color Highlight Python

### ISSUE THREAD

**Issue by xXxCryxXx** [thonny/thonny#2749]
The color disappears after saving the Projekt. if i try to restart the Programm it does not come back.

If i reinstall the Software the Colour comes back. 
(yes the file is named as .py)

System is Windows 11 64 bit Pro build 10.0.22621

Newest version of Thonny installed

**Comment by aivarannamaa** [comment]
Can you post a screenshot which shows this problem?

**Comment by jonathanvynck** [comment]
A lot of my students are experiencing this as well.
All syntax highlighting is just gone and the code looks like plain text.

**Comment by WayneDwane** [comment]
Yeah,i've had a similiar issue and i've found out that when i save the file i'm working on,it doesn't have the .py extension at the end. So try saving as > and then adding a .py to the end of the file. That fixed it for me. Your mileage might vary.

**Comment by WayneDwane** [comment]
> A lot of my students are experiencing this as well. All syntax highlighting is just gone and the code looks like plain text.

Also,do your students save their py files with numbers at the end? Like,for example,exercise 11 etc etc...?
That seems to be causing issues with the extension the file is saved as,hence the reason why a .py extension needs to be manually added.
